extensions/cpp/package.json

```diff
@@ -50,6 +50,13 @@
           "cpp"
         ],
         "configuration": "./language-configuration.json"
+      },
+      {
+        "id": "cuda-cpp",
+        "extensions": [
+          ".cu",
+          ".cuh"
+        ]
```

Comment:
Is a 'configuration' entry needed?

```
"configuration": "./language-configuration.json"
```

It looks like this provides some information on how to interpret comments and folding ranges.  You may be able to refer to the existing c/cpp config file.

Reply:
Hi Colen, we left that out because we have a follow-up PR to pull down the grammar and language configuration for cuda-cpp that would point it to a different configuration. But we could also set it to point to the C++ configuration for now and update it in the follow-up PR to the right path. I'm curious: Would it break something if it is left out?

Reply:
Added. Thank you.